qualtran/resource_counting/_bloq_counts.py

```diff
@@ -288,18 +287,6 @@ def compute(self, bloq: 'Bloq', get_callee_cost: Callable[['Bloq'], GateCounts])
         from qualtran.bloqs.bookkeeping._bookkeeping_bloq import _BookkeepingBloq
         from qualtran.bloqs.mcmt import And, MultiTargetCNOT
 
-        if self.legacy_shims:
-            legacy_val = bloq._t_complexity_()
-            if legacy_val is not NotImplemented:
-                warnings.warn(
-                    "Please migrate explicit cost annotations to the general "
-                    "`Bloq.my_static_costs` method override.",
-                    DeprecationWarning,
-                )
-                return GateCounts(
-                    t=legacy_val.t, clifford=legacy_val.clifford, rotation=legacy_val.rotations
-                )
-
```

Comment:
Please see that all tests pass without this shim; but I think I'll put this back in (using `getattr(bloq, '_t_complexity_', None)` for a bit of backwards compatibility support

Reply:
Latest commit puts this back in